packages/fxa-metrics-processor/src/config.ts

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import convict from 'convict';
+import fs from 'fs';
+import path from 'path';
+
+const conf = convict({
+  env: {
+    default: 'production',
+    doc: 'The current node.js environment',
+    env: 'NODE_ENV',
+    format: ['development', 'test', 'stage', 'production']
+  },
+  logging: {
+    app: { default: 'fxa-user-admin-server' },
+    fmt: {
+      default: 'heka',
+      env: 'LOGGING_FORMAT',
+      format: ['heka', 'pretty']
+    },
+    level: {
+      default: 'info',
+      env: 'LOG_LEVEL'
+    },
+    routes: {
+      enabled: {
+        default: true,
+        doc: 'Enable route logging. Set to false to trimming CI logs.',
+        env: 'ENABLE_ROUTE_LOGGING'
+      },
+      format: {
+        default: 'default_fxa',
+        format: ['default_fxa', 'dev_fxa', 'default', 'dev', 'short', 'tiny']
+      }
+    }
+  },
+  queues: {
+    pubsubProjectId: {
+      default: 'default-project',
+      doc: 'GCP Pub/Sub Project Id the queues are under',
+      env: 'PUBSUB_PROJECT_ID',
+      format: 'String'
+    },
+    rawEvents: {
+      default: 'fxa-metrics-raw-events',
+      doc: 'GCP Pub/Sub Queue for FxA Raw events',
```

Comment:
Would be good to be consistent about the use of "topic" vs. "queue" in the comment strings and the constant names.

Reply:
Well, technically instead of queue, it could be called subscription. Topic is separate though, as a topic can end up publishing to multiple subscriptions. Queue in this case is being used as a 'subscriber', while for publishing in PubSub we do not publish directly to a subscription, but a topic. See: https://cloud.google.com/pubsub/docs/overview#publisher-subscriber-relationships